test/auth_web/controllers/apikey_controller_test.exs

```diff
@@ -6,45 +6,53 @@ defmodule AuthWeb.ApikeyControllerTest do
   # alias AuthWeb.ApikeyController, as: Ctrl
   @email System.get_env("ADMIN_EMAIL")
   @create_attrs %{description: "some description", name: "some name", url: "some url"}
-  @update_attrs %{client_secret: "updated client sec", description: "some updated desc", name: "updated name", url: "surl"}
+  @update_attrs %{
+    client_secret: "updated client sec",
+    description: "some updated desc",
+    name: "updated name",
+    url: "surl"
+  }
   @invalid_attrs %{client_secret: nil, description: nil, key_id: nil, name: nil, url: nil}
 
-
   describe "Create an AUTH_API_KEY for a given person_id" do
     test "encrypt_encode/1 returns a base58 we can decrypt" do
       person_id = 1
       key = AuthWeb.ApikeyController.encrypt_encode(person_id)
       # |> IO.inspect(label: "key")
 
-      decrypted = key
-      |> Base58.decode
-      # |> IO.inspect(label: "decoded")
-      |> Fields.AES.decrypt()
-      # |> IO.inspect(label: "decrypted")
-      |> String.to_integer()
+      decrypted =
+        key
+        |> Base58.decode()
+        # |> IO.inspect(label: "decoded")
+        |> Fields.AES.decrypt()
+        # |> IO.inspect(label: "decrypted")
+        |> String.to_integer()
+
       # |> IO.inspect(label: "int")
 
       assert decrypted == person_id
     end
 
     test "decode_decrypt/1 reverses the operation of encrypt_encode/1" do
-      person_id = 4869234521
+      person_id = 4_869_234_521
```

Comment:
did `dialyxir` update the format of these numbers?

Reply:
It might have been `mix format` maybe

Reply:
Cool. Good to know they _both_ work. 👍 
`person_id` should be an unsigned int without the underscores because it's defined by PostgreSQL which does not use underscores in integers. 
I will update the code in the future to reflect this. 👍